assemble/conf/log4j2.properties

```diff
@@ -33,6 +33,9 @@ appender.console.layout.pattern = %style{%d{ISO8601}}{dim,cyan} %style{[}{red}%s
 logger.shellaudit.name = org.apache.accumulo.shell.Shell.audit
 logger.shellaudit.level = warn
 
+logger.shell.name = org.apache.accumulo.shell.Shell
```

Comment:
Why was this change made?

Reply:
Older versions of the shell supported a `--debug` option that would modify the log output. 
That flag has been deprecated in 2.x and no longer works. 

Now users have to correctly identify the name of the shell logger by either referencing source code or by modifying the layout.pattern and then set the logging level to achieve the same result as the original `--debug` flag. 

By adding the logger name in the log4j2 config, it reduces the level of effort needed for getting debug level logs. 

The expectation is that adding this should not change the current logging behavior of the shell as it was already emitting info level logs.

Reply:
> Older versions of the shell supported a --debug option that would modify the log output.

That makes sense.  Looking back at the 1.10 code that functionality used change the shell and core package logging settings.  Not sure if the core package should also be highlighted though. 

https://github.com/apache/accumulo/blob/47ac68d1a220a90bc80618f9684252b243df6b27/shell/src/main/java/org/apache/accumulo/shell/Shell.java#L1213

Reply:
If that's the behavior of the original shell then I can easily add a second line calling out the `org.apache.accumulo.core` logger specifically.